app/src/main/res/layout/fragment_product_detail.xml

```diff
@@ -249,73 +317,93 @@
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:layout_marginTop="@dimen/dp_10"
-                        android:contentDescription="@string/comments"
-                        app:layoutManager="LinearLayoutManager"
-                        app:recyclerAdapter="@{viewModel.commentAdapter}"
-                        />
+                        >
+                        <android.support.v7.widget.RecyclerView
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_marginTop="@dimen/dp_10"
+                            android:contentDescription="@string/comments"
+                            app:layoutManager="LinearLayoutManager"
+                            app:recyclerAdapter="@{viewModel.commentAdapter}"
+                            />
+                        <ProgressBar
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_gravity="center"
+                            android:visibility="@{viewModel.progressBarListCommentVisible ? View.VISIBLE : View.GONE}"
+                            style="@style/Base.Widget.AppCompat.ProgressBar"
+                            />
+                    </FrameLayout>
                     <TextView
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
                         android:layout_marginTop="@dimen/dp_5"
                         android:contentDescription="@string/see_more"
                         android:gravity="center"
                         android:text="@string/see_more"
-                        android:textColor="@color/color_black"
+                        android:textColor="@color/dark_gray"
+                        android:textSize="@dimen/sp_15"
+                        android:textStyle="bold"
                         />
-                    <ImageView
+                    <View
                         android:layout_width="match_parent"
-                        android:layout_height="@dimen/dp_90"
-                        android:contentDescription="@color/color_white"
+                        android:layout_height="@dimen/dp_55"
+                        android:background="@color/color_white"
                         />
                 </LinearLayout>
             </LinearLayout>
         </android.support.v4.widget.NestedScrollView>
         <RelativeLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+            android:layout_height="match_parent"
+            android:gravity="bottom"
+            tools:ignore="UselessLeaf"
             >
-            <LinearLayout
+            <RelativeLayout
```

Comment:
WARNING: [UselessParent] This `RelativeLayout` layout or its `RelativeLayout` parent is useless; transfer the `background` attribute to the other view
 A layout with children that has no siblings, is not a scrollview or a root layout, and does not have a background, can be removed and have its children moved directly into the parent for a flatter and more efficient layout hierarchy.